src/tests/EdFi.DataManagementService.Tests.E2E/Features/EqualityConstraintValidation.feature

```diff
@@ -1,13 +1,24 @@
 Feature: Equality Constraint Validation
               Equality constraints on the resource describe values that must be equal when posting a resource. An example of an equalityConstraint on bellSchedule:
     "equalityConstraints": [
-        {
-            "sourceJsonPath": "$.classPeriods[*].classPeriodReference.schoolId",
-            "targetJsonPath": "$.schoolReference.schoolId"
-        }
+    {
+    "sourceJsonPath": "$.classPeriods[*].classPeriodReference.schoolId",
+    "targetJsonPath": "$.schoolReference.schoolId"
+    }
     ]
-
+        @ignore
+        # See DMS-259 - Error inserting references
         Scenario: 01 Post a valid bell schedule no equality constraint violations.
+            Given the system has these "descriptors"
+                  | descriptorname                           | codeValue         | description       | namespace                                               | shortDescription  |
+                  | educationOrganizationCategoryDescriptors | School            | School            | uri://ed-fi.org/EducationOrganizationCategoryDescriptor | School            |
```

Comment:
Unless I misheard, I think @JBrenesSimpat said that he had built the system to automatically create descriptors. For example, when processing the School record below, the C# should detect the gradeLevelDescriptor and create it.

Reply:
> Unless I misheard, I think @JBrenesSimpat said that he had built the system to automatically create descriptors. For example, when processing the School record below, the C# should detect the gradeLevelDescriptor and create it.

I remember that something was said about this, but there is no logic to creating them automatically; it would be interesting to create a ticket for this.